test/base.test.js

```diff
@@ -223,7 +257,7 @@ test('should set new session cookie if expired', async (t) => {
     fastify.addHook('onRequest', (request, reply, done) => {
       request.sessionStore.set(DEFAULT_SESSION_ID, {
         cookie: {
-          expires: Date.now() + 1000
```

Comment:
This test was wrong and passed due to a bug, as explained here:
https://github.com/fastify/session/pull/121/files#r944682381